pandas/core/frame.py

```diff
@@ -2180,6 +2180,7 @@ def to_html(
         header=True,
         index=True,
         na_rep="NaN",
+        encoding="utf-8",
```

Comment:
Minor nit but can this default to `None` and handle the casing of that in the Formatter? That would keep the API consistent with the excel and csv formats

Reply:
Keep it `None` and let inner TableFormatter's `get_buffer_result` handle this.